src/tracking/events/BannerSubmitOnReturnEvent.ts

```diff
@@ -0,0 +1,14 @@
+import { TrackingEvent, TrackingFeatureName } from '@src/tracking/TrackingEvent';
+
+export class BannerSubmitOnReturnEvent implements TrackingEvent<void> {
```

Comment:
I think this new event needs some explanatory comment on what it is for. E.g. "We fire this event when the donor submits the banner form after previously closing another banner. The 'userChoice' contains the name of the close choice (see CloseChoice enum) that the donor previously used."

Reply:
When we finally have the new tracking schema we won't need this event at all, because we would put that information in as extra vars as part of the submit event